website/src/components/Notification/__snapshots__/Notification.test.jsx.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Notification Render Notification without any eror 1`] = `null`;
```

Comment:
I remember now. See the snapshot file. It is null.

Reply:
I see. This is because the notificationVisble value. I will try to fix it tomorrow.

Reply:
Now, we use enzyme testing instead of snapshot.